Sources/Crypto/Cipher/OpenSSLCipherAlgorithm.swift

```diff
@@ -0,0 +1,37 @@
+import CNIOOpenSSL
+
+/// OpenSSLCipherAlgorithm represents a common set of properties shared by
+/// OpenSSL cipher algorithms.
+public protocol OpenSSLCipherAlgorithm {
+    var c: UnsafePointer<EVP_CIPHER> { get }
+    init(c: UnsafePointer<EVP_CIPHER>)
+
+    var type: Int32 { get }
```

Comment:
All protocol members require doc block for 100% docs coverage. I recommend just copying the doc blocks you have from the implementation, then replacing the doc blocks there with:

```swift
/// See `OpenSSLCipherAlgorithm`
```

That way we prevent doc duplication and always point people to the most accurate docs.